website/docs/d/transit_verify.html.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: "vault"
+page_title: "Vault: vault_transit_verify data source"
+sidebar_current: "docs-vault-datasource-transit-verify"
+description: |-
+  Verify a signature using a Vault Transit signing key
+---
+
+# vault\_transit\_verify
+
+This is a data source which can be used to verify a signature using a Vault Transit key.
+
+## Example Usage
+
+```hcl
+data "vault_transit_verify" "test" {
+  backend     = "transit"
+  key         = "test"
+  signature   = "vault:v1:4kYRAVY/Q/6jjA3CT7HPhxKO+ru/4PhyGKBLRpn9DSeT99McPXEk302NXtzCzsvbSOZPif7f32tlr58iYoxjCQ=="
+  input       = "aGVsbG8gd29ybGQ="
+}
+```
+
+## Argument Reference
+
+Each document configuration may have one or more `rule` blocks, which each accept the following arguments:
+
```

Comment:
Missing `path` and `namespace` docs, and I think the example needs to be updated to use `path` not `backend`